dragonphy/__init__.py

```diff
@@ -22,7 +22,7 @@
 from .cmdparser		 import CmdLineParser
 from .packager       import Packager
 from .config 		 import Configuration
-from .tester     	 import DragonTester
+#from .tester     	 import DragonTester
```

Comment:
Hmm, why is this commented out?  I think this would break a number of simulation-based tests.